src/main/io/osd.c

```diff
@@ -153,6 +153,13 @@ static char osdGetAltitudeSymbol()
     }
 }
 
+static char osdGetBatterySymbol(int cellVoltage)
+{
+    /* Calculate current battery cell voltage as a percentage of range */
+    int voltPercent = ((cellVoltage - (batteryConfig()->vbatmincellvoltage * 10)) * 7) / ((batteryConfig()->vbatmaxcellvoltage - batteryConfig()->vbatmincellvoltage) * 10);
```

Comment:
I think that's not wrong at all. Users doing long distance flying will probably want to set up their battery warning so that they get alerted when there's still plenty of battery left, in order to ensure they have enough for trip back to the airfield.